.pydocstyle

```diff
@@ -1,5 +1,6 @@
 [pydocstyle]
-ignore = D104,  # Missing docstring in public package
+ignore = D103,  # Missing docstring in public function
+         D104,  # Missing docstring in public package
```

Comment:
Is it better to have one nonsense docstring rather than forgetting to put them where they are needed?

Reply:
Done

Reply:
It seems we're taking a stricter approach for docstrings in our packages than we are in our apps. Maybe we should also take the opportunity to use the stricter approach in new apps too. Via 3 might still be saveable. h and lms are probably too far gone: too much stuff without docstrings already

Reply:
Seems like a good green fields time to try. It honestly doesn't take _that_ long to go through and add them, but if you are doing loads at once you tend to end up writing nonsense before long. It's marginal benefit at best. 

We could try removing the exclusion temporarily to see how many there are. Fix a few and put it back. Rinse repeat until done.